bin/cloudgov/audit/db-integrity

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+#
+# Requires sqlvalidator, run pip install sqlvalidator if this doesn't work.
+#
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+if [ -f $SCRIPT_DIR/../../deploy/includes ]; then
+  . $SCRIPT_DIR/../../deploy/includes
+else
+   echo Cannot find $SCRIPT_DIR/../../deploy/includes
+   exit 1
+fi
+
+# just testing?
+if [ x$1 = x"--dryrun" ]; then
+  export echo=echo
+  shift
+fi
+
+SPACE=${1:-please-provide-space-name-as-first-argument}
+SPACE=$(echo "$SPACE" | tr '[:upper:]' '[:lower:]')
+assertCurSpace $SPACE
+shift
+
+echo "Testing static site integrity."
+
+echo "Getting s3 access."
+source bin/cloudgov/get-s3-access storage >/dev/null 2>&1
+
+echo "Getting three random backup tags from s3."
+DEPLOY_TAGS=$(aws s3 ls s3://"$S3_BUCKET"/db-backup/ | awk '{ print $4 }' |  sort -R | tail -n 3 )
+
+echo "-----------------------------------"
+echo "$DEPLOY_TAGS" | while read -r DEPLOY_TAG ; do
+  echo "Downloading db: $DEPLOY_TAG"
+  aws s3 cp --only-show-errors s3://"$S3_BUCKET"/db-backup/"$DEPLOY_TAG" "$DEPLOY_TAG"
+  gunzip "$DEPLOY_TAG"
+  sleep 5
+  SQL_FILE=$(echo "$DEPLOY_TAG" | sed 's/.gz$//')
+
+  echo "Testing: $SQL_FILE"
+
+  size=$(du -hs "$SQL_FILE")
+  echo "SQL file size: $(echo "$size" | awk '{print $1}')"
+
+  echo "Validating SQL. This will take a while."
+  validate=$(sqlvalidator --verbose-validate "$SQL_FILE")
+  if [ -z "$validate" ]; then
+    echo "SQL file is valid."
```

Comment:
Let's move the `rm $SQL_FILE` line up here (that is, don't delete it if it's invalid; I might want to look at it!)